### ISSUE TITLE
sdk/client/resourcemanager: `databricks` LROs contain an empty string payload

### ISSUE THREAD

**Issue by tombuildsstuff** [hashicorp/go-azure-sdk#351]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Community Note

* Please vote on this issue by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Service Used

`databricks`

### API Versions Used

`2023-02-01`

### Description

The Delete LRO for a DataBricks Workspace returns an empty string response rather than encoded JSON, whilst we ignore this response body today we should ensure this is supported:

```
DELETE /subscriptions/{subscription}/resourceGroups/{group1}/providers/Microsoft.Databricks/workspaces/{workspace1}/virtualNetworkPeerings/databricks-vnet-peer?api-version=2023-02-01 HTTP/1.1
Host: management.azure.com
User-Agent: Go/go1.19.6 (amd64-linux) go-autorest/v14.2.1 hashicorp/go-azure-sdk/vnetpeering/2023-02-01 HashiCorp Terraform/1.3.9 (+https://www.terraform.io/) Terraform Plugin SDK/2.10.1 terraform-provider-azurerm/dev pid-222c6c49-1b0a-5959-a213-6608f9eb8820
Content-Type: application/json; charset=utf-8
X-Ms-Correlation-Request-Id: 5aa7981b-127c-16f5-0304-25946d291574
Accept-Encoding: gzip
```

```
[DEBUG] provider.terraform-provider-azurerm: AzureRM Response for https://management.azure.com/subscriptions/{subscription}/resourceGroups/{group1}/providers/Microsoft.Databricks/workspaces/{workspace1}/virtualNetworkPeerings/databricks-vnet-peer?api-version=2023-02-01:
HTTP/2.0 202 Accepted
Content-Length: 3
Cache-Control: no-cache
Content-Type: application/json
Date: Mon, 06 Mar 2023 21:21:23 GMT
Expires: -1
Location: https://management.azure.com/subscriptions/{subscription}/providers/Microsoft.Network/locations/westeurope/operationResults/d6e4b8b8-51d2-4e98-aaf2-0e38a7fce24a?api-version=2018-08-01
Pragma: no-cache
Strict-Transport-Security: max-age=31536000; includeSubDomains
X-Content-Type-Options: nosniff
X-Frame-Options: SAMEORIGIN
X-Ms-Correlation-Request-Id: 5aa7981b-127c-16f5-0304-25946d291574
X-Ms-Ratelimit-Remaining-Subscription-Deletes: 14999
X-Ms-Request-Id:
X-Ms-Routing-Request-Id: WESTUS2:20230306T212124Z:672eae63-5d76-44cc-a756-56717e92ec7b
X-Xss-Protection: 1; mode=block

""
```

**Comment by WodansSon** [comment]
Related [PR](https://github.com/hashicorp/terraform-provider-azurerm/pull/20728) hitting this issue.

**Comment by tombuildsstuff** [comment]
This is now available in `v0.20230317.1100159` of `github.com/hashicorp/go-azure-sdk` / fixed by https://github.com/hashicorp/pandora/pull/2241
